File: android/src/main/java/com/tozny/plugin_tozny/RealmConfigSerializer.java
package com.tozny.plugin_tozny;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.net.URI;

public class RealmConfigSerializer {
    @JsonProperty("realm_name")
    public String realmName;

    @JsonProperty("app_name")
    public String appName;

    @JsonProperty("broker_target_url")
    public URI brokerTargetURL;

    @JsonProperty("api_url")
    public URI apiURL;
}

File: ios/Classes/RealmConfigSerializer.swift
public class RealmConfigSerializer: Codable {
    let realmName: String
    let appName: String
    let brokerTargetURL: String
    let apiURL: String
    
    enum CodingKeys: String, CodingKey {
        case realmName = "realm_name"
        case appName = "app_name"
        case brokerTargetURL = "broker_target_url"
        case apiURL = "api_url"
    }
}
